curve_apps/utils.py

```diff
@@ -23,6 +24,43 @@
 from curve_apps.trend_lines.params import TrendLineDetectionParameters
 
 
+def interp_to_grid(
+    entity: ObjectBase, values: np.ndarray, resolution, max_distance
+) -> tuple[list[np.ndarray], np.ndarray]:
+    """
+    Interpolate values into a regular grid based on entity locations.
+
+    :param entity: Geoh5py object with locations data.
+    :param values: Data to be interpolated to grid.
+    :param resolution: Grid resolution
+    :param max_distance: Maximum distance used in weighted average.
+    """
+
+    grid = []
+    for i in np.arange(2):
```

Comment:
Surprized that linters didn't complain about single letter variable.

Reply:
na na na boo boo linter.  I think it's totally reasonable to have single letter iterables